### ISSUE TITLE
Bug: Generating a Drop-Down List With an Empty Placeholder

### ISSUE THREAD

**Issue by yasinkocak** [LaravelCollective/html#124]
When u use on template following:
{!! Form::select('size', array('L' => 'Large', 'S' => 'Small'), null, ['placeholder' => 'Pick a size...']) !!}

This not generate empty placeholder.

**Comment by arthurkirkosa** [comment]
This is what I get generated

```html
<select name="size">
  <option selected="selected" value="">Pick a size...</option>
  <option value="L">Large</option>
  <option value="S">Small</option>
</select>
```

```js
{
            "name": "laravelcollective/html",
            "version": "v5.1.9",
}
```

**Comment by yasinkocak** [comment]
Hi,

I get this.

```
<select name="size" placeholder="Pick a size...">
  <option value="L">Large</option>
  <option value="S">Small</option>
</select>
```

```
{
            "name": "laravelcollective/html",
            "version": "v5.0",
}
```

**Comment by adamgoose** [comment]
The feature was released with 5.1. It is not available in the 5.0 version.

**Comment by arthurkirkosa** [comment]
Upgrade; most likely it was added in 5.1
